retrieving.md

```diff
@@ -11,7 +11,7 @@ There are several idiomatic operations to retrieve results from the datastore.
 1. Execute a query that returns a single row. There is a shortcut for this special case.
 
 Go's `database/sql` function names are significant. **If a function name
-includes `Query`, it is designed to ask a question of the database, and will
+includes `Query`, it is designed to ask a question to the database, and will
```

Comment:
This is probably a dialectical preference, and may not be universal, but "ask a question of" is considered proper and sensible in the English I'm familiar with, whereas "ask a question to" is odd-sounding. It's probably similar to the various usages of different: "x is different from y" vs "x is different than y" vs "x is different to y" are all "normal" in various dialects. Perhaps there is an alternative phrasing that avoids this issue, such as "it is designed to ask the database a question" or just "designed to ask a question". What do you think?

Reply:
To me, `ask a question of the database` sounded incorrect. But since there are multiple versions of English and people around the world speak different forms of English, we might to keep it as it is.

`it is designed to ask the database a question` also sounds fine but `it is designed to ask a quesiton of the database` sounds more appropriate. I would like to keep it as it is and close this PR.

What do you think?

Reply:
Sounds good to me!